canonical-webteam/django-views
# Match templates case-insensitively

Helps with https://github.com/canonical-websites/www.ubuntu.com/issues/4448

QA
--

Check you agree that the test tests properly, and pass.

Then test out this PR: https://github.com/canonical-websites/www.ubuntu.com/pull/4486
